src/getting_started.md

```diff
@@ -195,6 +195,69 @@ The board should appear as a regular serial device (e.g.
 `/dev/tty.usbserial-c098e5130006` on my Mac or `/dev/ttyUSB0` on my Linux box).
 This may require some setup, see the "one-time fixups" box.
 
+### On Windows Subsystem for Linux (WSL)
+
+Programming JLink devices with Tock in WSL:
+
+Trying to program an nRF52840DK with WSL can be a little tricky because WSL2
+abstracts away low level access for USB devices. WSL1 is unable to find JLink
+devices even if you have JLink installed because of the USB abstraction. To get
```

Comment:
Does this mean both WS1 and WS2 don't work, but for different reasons, and the fix works for both of them?

Reply:
No. WSL 1 does not provide access to hardware devices. The USBIP tool is only built for WSL2. I should replace WSL2 and WSL1 here with just WSL. Microsoft enabled support since WSL kernel version 5.10.60.1. 

This fix requires users to set their WSL version to 2, I mention this in step 5.

Also, I remember doing this on Windows 10 before, but the steps are probably different.

Reply:
I think I have addressed this better now.